tests/integration/verticalsearch.ts

```diff
@@ -169,6 +169,24 @@ it('answers.setVerticalLimit sets the vertical limit when a number is passed to
   expect(answers.state.vertical.limit).toEqual(7);
 });
 
+it('vertical searches re-use queryId when offset is non-0', async () => {
```

Comment:
Maybe this is already covered by another test, but should we have a test where no query id is passed? Like offset = 0 and input query is different?

Reply:
Sure, added. Also fixed the tests overall (queryId wasn't getting generated)